inngest/src/main/kotlin/com/inngest/Function.kt

```diff
@@ -82,8 +82,11 @@ internal class InternalFunctionConfig
         val throttle: Throttle? = null,
         @Json(serializeNull = false)
         val debounce: Debounce? = null,
+        @Json(serializeNull = false)
```

Comment:
I missed this when resolving merge conflicts for https://github.com/inngest/inngest-kt/pull/71